lighthouse-core/audits/seo/hreflang.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+const LinkHeader = require('http-link-header');
+const VALID_LANGS = importValidLangs();
+const LINK_HEADER = 'link';
+const NO_LANGUAGE = 'x-default';
+
+/**
+ * Import list of valid languages from axe core without including whole axe-core package
+ */
+function importValidLangs() {
+  const axeCache = global.axe;
+  global.axe = {utils: {}};
+  require('axe-core/lib/commons/utils/valid-langs.js');
+  const validLangs = global.axe.utils.validLangs();
+  global.axe = axeCache;
+
+  return validLangs;
+}
+
+/**
+ * @param {String} hreflang 
```

Comment:
lowercase `string`/`boolean` for all of these (this will soon [actually be checked](https://github.com/GoogleChrome/lighthouse/pull/3747)! :)